packages/core/template_project/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+    Copyright 2019 Google Inc. All Rights Reserved.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+    <color name="shortcut_background">#F5F5F5</color>
```

Comment:
Is there any reason we might want to allow developers customising the colour? If that's the case, we may want to consider defining this in `app/build.gradle`.

Reply:
No, this is the [standard design on Android](https://commondatastorage.googleapis.com/androiddevelopers/shareables/design/app-shortcuts-design-guidelines.pdf) and shouldn't be changed.